editor/src/components/canvas/controls/size-box.tsx

```diff
@@ -302,7 +302,7 @@ interface ResizePointProps {
   testID: string
 }
 
-class ResizePoint extends React.Component<ResizePointProps, {}> {
+class ResizePoint extends React.Component<ResizePointProps, unknown> {
```

Comment:
```suggestion
class ResizePoint extends React.Component<ResizePointProps> {
```

based on the diff, this unknown can be omitted